### DISCUSSION TITLE
Avalonia Port

### DISCUSSION THREAD

**Discussion by furesoft** [JamesnetGroup/navigationbar#2] [Show and tell]
Hi, 

I am working on a Avalonia Port, Here is the current state

![image](https://github.com/vickyqu115/navigationbar/assets/4117602/5fccef69-f7ca-4f1b-b7a3-62405fdb798b)

Some things are hard to implement and for me I do not have any knowledge about to port your animations

here is the source: https://github.com/furesoft/NavigationBar

PS: the avalonia version is not bound to 5 items.

**Comment by furesoft** [comment]
a little bit hacky but it works now

![image](https://github.com/vickyqu115/navigationbar/assets/4117602/4edd81b8-22bd-4a7f-9c82-3233356236a6)
![image](https://github.com/vickyqu115/navigationbar/assets/4117602/46be7f50-b53e-4b9d-be07-516269fd1d7a)

**Comment by jamesnet214** [comment]
@furesoft Oh! That's really wonderful. I hope many people get to see the work you've shared. Thank you so much for responding to my request.

**Comment by vickyqu115** [comment]
I haven't tried AvaloniaUI yet, but comparing it to the WPF version, it seems there are many similarities as well as some differences, which I find fascinating and fun!👍🏻
Thank you for sharing!!
